homeassistant/components/coinbase/strings.json

```diff
@@ -6,7 +6,7 @@
         "description": "Please enter the details of your API key as provided by Coinbase.",
         "data": {
           "api_key": "[%key:common::config_flow::data::api_key%]",
-          "api_token": "API Secret"
+          "api_token": "[%key:common::config_flow::data::api_token%]"
```

Comment:
Could this be here to match terminology of Coinbase?

Reply:
Checked, you are right. Reverted, thanks.